docs/imviz/export_data.rst

```diff
@@ -111,3 +111,10 @@ The columns are as follow:
 
 Once you have the results in a table, you can further manipulated them as
 documented in :ref:`astropy:astropy-table`.
+
+Markers Table
+=============
+
+All mouseover information in the :ref:`markers plugin <markers-plugin>` can be exported to an
+:ref:`astropy table <astropy:astropy-table>`
+by calling :meth:`~jdaviz.core.template_mixin.TableMixin.export_table` (see :ref:`plugin-apis`).
```

Comment:
I still feel like we need to give example code here. A typical user won't have the patience to click through several API doc pages to find it.

```suggestion
by calling :meth:`~jdaviz.core.template_mixin.TableMixin.export_table` (see :ref:`plugin-apis`):

.. code-block:: python

    markers_plugin = imviz.plugins["Markers"]
    markers_table = markers_plugin.export_table()
```